### DISCUSSION TITLE
Pixijs 6.5.1+ angular13 library not working

### DISCUSSION THREAD

**Discussion by thangaraj80** [pixijs/pixijs#8553] [Q&A]
I am building a library that uses Pixijs 6.5.1 and it works fine when we 'ng serve' as the individual library and it loads the sprite.  while I try to integrate the library into another project sprite is not loading and no errors. Do I need to modify compiler options in Angular 13?

**Comment by bigtimebuddy** [comment]
Could you provide a simple boilerplate example so we can see what the issue is?

**Reply by Julien-Marcou** [reply]
I have a working repository with Angular 13 and PixiJS 6, you can take a look at it: https://github.com/julien-marcou/puzzle

**Reply by thangaraj80** [reply]
> Could you provide a simple boilerplate example so we can see what the issue is?

I created the library to load the image in Pixi Canvas, it's working fine in the testbed library. 

After building the library in internal_npm, try to integrate another project. it stops working.

I check the code chunked file of that library in the project and found **return new Observable**  from rxjs not fully execute it returns at the first line itself e.g


```
 return new Observable((observer: Observer<any>) => {
   const newId = URI(id).protocol('db').toString();   -----> its returns here itself

     let pouchRev: any;
      if (Object.keys(this.deferreds).indexOf(URI(id).protocol('db').toString()) === -1) {    -----> not checking this conditions
```

**Reply by Julien-Marcou** [reply]
I never encountered an error with Observable and PixiJS in an Angular project, this doesn't seem to be an error with PixiJS.

Have you tried completely deleting the `node_modules` folder and the `package-lock.json` file, then making a new `npm install` to be sure all your dependencies are properly resolved/installed?
